### ISSUE TITLE
install dependency from path fails with  `Error: ENOENT: no such file or directory` error

### ISSUE THREAD

**Issue by ItsCosmo** [PixnBits/githook-scripts#13]
If installing a dependency from a path, e.g. "git+ssh:some.git.repo",if the dependency uses githooks, the install will fail with a message 
```
 Error: ENOENT: no such file or directory, open '/Users/CURRENT USER/.npm/_cacache/tmp/git-clone-cbac6cce/.git/hooks/pre-commit'
```

**Comment by PixnBits** [comment]
evidently saying "_maybe_ fixes" is the same as "fixes" to GitHub 😓 Sorry

**Comment by ItsCosmo** [comment]
looks like this was addressed by PR #26. Thanks!
